src/lib.rs

```diff
@@ -1,4 +1,6 @@
+#![allow(dead_code)]
```

Comment:
I know this silences those warnings, but I'd rather see them for now. This way, once we write the Template module, we'll know what is and isn't being used for realz